modules/swagger-parser-v3/src/test/java/io/swagger/v3/parser/util/RefUtilsTest.java

```diff
@@ -220,23 +222,27 @@ public void testReadExternalRef_InternalFormat(@Injectable final List<Authorizat
         final String file = "#/defintiions/foo";
 
         try {
-            RefUtils.readExternalRef(file, RefFormat.INTERNAL, auths, null);
+            RefUtils.readExternalRef(file, RefFormat.INTERNAL, auths, null, null);
             fail("Should have thrown an exception");
         } catch (RuntimeException e) {
 
         }
     }
 
-    @Test
+    /*@Test
```

Comment:
If this test is no longer relevant, remove it; otherwise, uncomment and fix it.

Reply:
Thanks! I'll review and make the changes. This a work in progress